### PR TITLE
improved .md files

### PR DESCRIPTION
**Pull request by DeanLemans**
made white-space consistent.

deleted NOTES_CLI.MD, NOTES_XMRIG-PROXY.md and TODO_XMRvsBeast.md.

### CODE REVIEW

File: ARCHITECTURE.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Gupaxx ARCHITECTURE
+# **Gupaxx ARCHITECTURE**
```
Comment (Cyrix126) [inline ARCHITECTURE.md:1]: No need to make header 1&2 bold.

File: IDEAS.md [hunk 2]
```diff
@@ -1,120 +1,173 @@
...
 
-## More Decentralized
+## **Making Gupaxx more Decentralized**
 ### Synchronize source code repository on p2p network
-Github is proprietary. If Gupaxx aims to be free, it should not be only available on this platform and we should explore options to get github free.  
-We can use [Radicle](https://radicle.xyz/) to get Gupaxx on a p2p collaboration stack.  
-The code, issues and PR could be synchronized with Github.  
+- Github is proprietary. If Gupaxx aims to be free software, it should not be only available on this proprietary platform and we should explore options to be github free.  
+- We can use [Radicle](https://radicle.xyz/) to get Gupaxx on a p2p collaboration stack.  
+- We can use [Codeberg](https://codeberg.org/) as an open source alternative for Github.
```
Comment (Cyrix126) [inline IDEAS.md:11]: Please keep commits for grammar/formatting separated with new ideas.

File: IDEAS.md [hunk 3]
```diff
@@ -1,120 +1,173 @@
...
+> The code, issues and PR could then also be synchronized with Github.  
+(We could create mirrors between Github, Radicle and/or Codeberg)
+
 ### Integrate a P2Pool compatible Nodes crawler
 To get rid of integrating nodes list, we can include a crawler like monero.fail inside Gupaxx.
+
```
Comment (Cyrix126) [inline IDEAS.md:17]: Space are not needed in markdown format if there is a header in between.

**Review by Cyrix126 — CHANGES_REQUESTED** [review]
CLI notes and Proxy notes markdown should just be deleted.
Space between headers and text is not needed.
Bold headers are not needed.
New ideas should be in separate PR or issue.

File: ARCHITECTURE.md (hunk 1 above)
Comment (DeanLemans) [reply ARCHITECTURE.md:1]: I disagree, I think that making the header of level 1 and 2 bold, creates a better readability.

if you still think that its better I will remove them.

**Comment by DeanLemans** [comment]
> CLI notes and Proxy notes markdown should just be deleted. 

should I also delete the TODO_XMRvsBeast.md file?

File: IDEAS.md (hunk 3 above)
Comment (DeanLemans) [reply IDEAS.md]: https://github.com/Cyrix126/gupaxx/pull/41/commits/04d358f8a516f847f4378ff23a1a7a34d488a325

File: IDEAS.md (hunk 2 above)
Comment (DeanLemans) [reply IDEAS.md]: will create seperate pr for grammar and formatting when this pr is merged/accepted.

**Comment by Cyrix126** [comment]
> > CLI notes and Proxy notes markdown should just be deleted.
> 
> should I also delete the TODO_XMRvsBeast.md file?
That file mostly resume what I did for the bounty, but in the same time one could just look at release or even commits before the bounty was done to see what work has been done specifically for it. So yes let's delete it.

**Review by Cyrix126 — CHANGES_REQUESTED** [review]
Still need to remove the bold text in headers. See my previous comment to the reason why.

**Review by Cyrix126 — APPROVED** [review]

**Comment by Cyrix126** [comment]
Thanks for theses improvements ! 
Feel free to make another PR for ideas.  I need to take the time to think if they should be converted to issues but I will gladly add yours to the file in the mean time.

**Pull request merged**